pkgs/os-specific/linux/ryzen-monitor-ng/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "ryzen_monitor_ng";
+  version = "2.0.5";
```

Comment:
```suggestion
  version = "unstable-2023-11-05";
```

Reply:
plasmin’s repo doesn't have any tag, it makes it difficult to find out the latest versio. I don't know if it should have a version from a more original repo, but don't oppose it.

Reply:
I guess I am using the fork as a small set of patches.

I can revert to using the orginal repo, and pull in the patches from the forks if that seems better to you.

Reply:
Then the version would be from the original repo, but it would still be unstable.